packages/frontend/src/features/sqlRunner/components/ContentPanel.tsx

```diff
@@ -94,8 +94,8 @@ export const ContentPanel: FC<Props> = ({
         isLoading,
     } = useSqlQueryRun({
         onSuccess: (data) => {
-            if (data?.results) {
```

Comment:
I might have changed this type this morning. Just FYI. So it *might* need a type guard or we can change how this works.